viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -2,23 +2,27 @@
  * Copyright 2023 Cognite AS
  */
 
-import { CanvasTexture, Frustum, Matrix4, Texture, Vector3 } from 'three';
+import { CanvasTexture, Frustum, Matrix4, Sprite, SpriteMaterial, Texture, Vector2, Vector3 } from 'three';
 import { BeforeSceneRenderedDelegate, EventTrigger, SceneHandler } from '@reveal/utilities';
-import { Image360Icon } from './Image360Icon';
-import { InstancedIconSprite } from './InstancedIconSprite';
-import { IconOctree } from './IconOctree';
+import { IconOctree, Overlay3DIcon } from '@reveal/3d-overlays';
 import clamp from 'lodash/clamp';
+import { Image360PointsObject } from './Image360PointsObject';
 
 export type IconCullingScheme = 'clustered' | 'proximity';
 
+export type IconsOptions = {
+  platformMaxPointsSize?: number;
+};
+
 export class IconCollection {
   private readonly MIN_PIXEL_SIZE = 16;
-  private readonly MAX_PIXEL_SIZE = 256;
+  private readonly DEFAULT_MAX_PIXEL_SIZE = 256;
+  private readonly MAX_PIXEL_SIZE: number;
```

Comment:
Don't know if I'd have this in all-caps when it's not really a constant (just `readonly`)

Reply:
Changed with the same convention as in `DefaultCameraManager`. Constants are now static and `MAX_PIXEL_SIZE` is just camel case.